ui/apps/platform/src/hooks/useAnalytics.ts

```diff
@@ -61,6 +74,42 @@ type AnalyticsEvent =
     | typeof CLUSTER_CREATED
     | typeof INVITE_USERS_MODAL_OPENED
     | typeof INVITE_USERS_SUBMITTED
+    /** Tracks each time a cluster level simulator is opened on Network Graph */
+    | {
+          event: typeof CLUSTER_LEVEL_SIMULATOR_OPENED;
+          properties: {
+              cluster: string;
+              namespaces: string;
+              deployments: string;
```

Comment:
will this pass through the literal cluster/ns/deployment names to amplitude? if so, is that the behavior that we want?

Reply:
Another good question. We could wire up a lookup from names to UUID, to obfuscate the data, or only pass through counts (for example, 1 NS and 0 Deployments).

I was assuming that entity names were not private data. (In other contexts, we allow users without cluster permission to see the names of clusters, in order to perform necessary business tasks, on a similar assumption.)
However, maybe it would be safer to start with counts and then add info later, when we've had time to vet the use-case.

Reply:
+1 FWIW, we intentionally went back and changed the VM 2.0 analytics to exclude cluster/namespace/deployment names.

Reply:
As a guideline for names: return only UI **feature** names, not customer **entity** names.

If potential UI/UX questions include which combinations of one versus multiple for filtering, then the counts are the data. If limited data collection were not enough reason, it an extra step to extract counts from arrays of names or joined names seems impractical to analyze the analytics.